crates/ty_python_semantic/resources/mdtest/assignment/annotations.md

```diff
@@ -259,7 +259,7 @@ class Color(Enum):
     RED = "red"
 
 f: dict[list[Literal[1]], list[Literal[Color.RED]]] = {[1]: [Color.RED, Color.RED]}
-reveal_type(f)  # revealed: dict[list[Literal[1]], list[Literal[Color.RED]]]
+reveal_type(f)  # revealed: dict[list[Literal[1]], list[Color]]
```

Comment:
Our previous support for enum member literals in `typing.Literal` did not implement the simplification of a single-member enum literal type to the overall enum type. Since we do implement this simplification otherwise, I thought it more consistent to do it in this case also. (And it falls out from the implementation, which now simply uses general type inference rather than special handling for enum members.)